README.md

```diff
@@ -270,6 +270,7 @@ Parses the info from the `hltv.org/stats/teams/*` page (4 requests)
 | Option |  Type  | Default value | Description |
 | :----: | :----: | :-----------: | :---------: |
 |   id   | number |       -       | The team id |
+|   currentRoster   | boolean |       false       | Only Current Roster |
```

Comment:
Can you change it to `currentRosterOnly`? I feel like it's a bit more descriptive.

Reply:
Yep!